package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dell/clarity-react",
-    "version": "0.21.1",
+    "version": "0.21.2",
```

Comment:
Should this be 0.22.0?

Reply:
@maheshkulkarni4 and @ben should we update minor version here or major ? Please provide your thoughts.

Reply:
I guess we are adding functionality -I'll bump to 0.22.0.